### ISSUE TITLE
Where does this come from? self.loss = gluon.loss.RNNTLoss(blank_label=blank)

### ISSUE THREAD

**Issue by rzcwade** [HawkAaron/RNN-Transducer#2]
Hello,

I am trying to run your code and I discovered that you used gluon.loss.RNNTLoss() which does not exist. Could you clarify on this function? Did you mean from warprnnt_pytorch import RNNTLoss?

Thanks.

**Comment by HawkAaron** [comment]
This is a mxnet version, so you need to use mxnet rnnt loss, which is in https://github.com/HawkAaron/mxnet-transducer/tree/add_network

**Comment by rzcwade** [comment]
Thanks, @HawkAaron !
